frontend/packages/operator-lifecycle-manager/src/utils/consolePluginFormGroup.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
```

Comment:
nit: In this package, we've been kebab casing the file names (e.g., console-plugin-form-group.tsx)